db/models/round.py

```diff
@@ -82,3 +82,6 @@ class Round(BaseModel):
     feedback_survey_config = Column(
         "feedback_survey_config", JSON(none_as_null=True), nullable=True, unique=False
     )
+    eligibility_config = Column(
+        "eligibility_config", JSON(none_as_null=True), nullable=True, unique=False
```

Comment:
Should there not be a boolean value in here for whether eligibility is being used which defaults to false? I think there's a separate ticket looking to deal with the config based on the results of the design team

Reply:
Yes there is, inside this json column.
The full config from the design team (like what questions to ask) will be turned into form jsons - this is more about how we present/use that eligibility stuff.
So I could have added a single boolean column, but incase they want to configure more things about eligibility in the future (maybe they want eligibility on or off, and maybe sometimes they want a failure in eligibility to prevent them from applying, maybe sometimes it's just an advisory thing) I've made it json so it can hold multiple config values.
We had a thing with feedback where we ended up just adding a ton of columns to this table, so decided to use json for config in the future to cope with multiple values.
Sorry for the long reply!

Reply:
No worries at all! Long response is good, that all makes sense to me